recipe/meta.yaml

```diff
@@ -23,9 +23,9 @@ requirements:
   - bob.core
   - bob.io.base
   - giflib
-  - libpng
-  - jpeg
-  - libtiff
+  - libpng 1.6*
+  - jpeg 9*
+  - libtiff 4.0.*
```

Comment:
Different from the hdf5 comment I made [here](https://github.com/conda-forge/bob.io.base-feedstock/pull/1/files#r60215227), the pinning done here is very welcomed.